browser/src/errormessages.js

```diff
@@ -40,13 +40,13 @@ if (window.ThisIsAMobileApp) {
 	};
 } else {
 	errorMessages.storage = {
-		loadfailed: _('Failed to read document from storage. Please contact your storage server (%storageserver) administrator.'),
+		loadfailed: _('Failed to read document from storage. Please try to load the document again.'),
 		savediskfull: _('Save failed due to no storage space left. Document will now be read-only. Please make sure enough disk space is available and try to save again.'),
-		savetoolarge: _('Save failed because the document is too large or disk quota exceeded. Document will now be read-only but you may still download it now to preserve a copy locally. Please contact the server (%storageserver) administrator to resolve the issue.'),
+		savetoolarge: _('Save failed because the document size exceeds the remaining storage space. The document will now be read-only but you may still download it now to preserve a copy locally.'),
```

Comment:
Agreed with @thebearon. Note also that some servers reply with the same error code for exceeding quota limit (for the user), running out of disk space, and for the document being larger than the limit (even if the quota allows for said size and the storage has plenty space).

So, we cannot assume that the "document size exceeds the remaining storage space". The original wording was more ambiguous intentionally.

Otherwise, reasonable to remove the "contact the server" part.